instances/atlas/parcellationEntity/JBA/JBA_temporal-insula.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_temporal-insula",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_temporal-lobe"
+        }
+    ],
+    "hasVersion": null,
+    "lookupLabel": "JBA_temporal-insula",
+    "name": "temporal-insula",
```

Comment:
I think the name should be without "-", no?

Reply:
It is as the json hartmut gave us

https://gin.g-node.org/FZJ-INM1-BDA/20210615-julichbrain-v2.9.0/src/master/workingdir/hbp-d000006_julichbrain-cytoatlas_dev/JulichBrainCytoAtlas/julichbrain-ontology_29.0.json

(require authentication/authorization, what is your gin user name? I will add you?)

we can certainly change it, just say the word.

Reply:
well if this is what he defines leave it for now. we can discuss refinements with the Julich-Brain team later.